charts/posthog/values.yaml

```diff
@@ -745,6 +746,33 @@ externalStatsd:
   port:
 
 
+###
+###
+### ---- prometheus-kafka-exporter ----
+###
+###
+prometheus-kafka-exporter:
+  # -- Whether to install the `prometheus-kafka-exporter` or not.
+  enabled: false
+
+  # -- We want to pin to image tag `v1.4.2` as it is currently the only
+  # available version working on Apple M1 (otherwise we break local development).
+  #
+  # TODO: remove the override once `prometheus-kafka-exporter` will default to this version.
+  #
+  image:
+    tag: v1.4.2
+
+  # -- Map of annotations to add to the pods.
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9308"
```

Comment:
Where does this port number come from. I can see it is used as the default service port, but is it guaranteed to be the pod target port also?

Reply:
The port number is coming from the exporter binary itself